e2e-tests/create-large-test-files.sh

```diff
@@ -10,4 +10,3 @@ dd if=/dev/zero of=cypress/fixtures/appeal-statement-valid-max-size.png bs=$MAX_
 dd if=/dev/zero of=cypress/fixtures/appeal-statement-invalid-too-big.png bs=$TOO_BIG count=1
```

Comment:
not sure it's useful, maybe a rerun would have be sufficient but I think that it's this script which sometimes make the npm ci failing. e2e-tests is the last of the folder builded so we might had silent build errors.